### PR TITLE
Document that concatenation is enabled by default

### PR DESCRIPTION
**Pull request by rluba**
I recently tried to add `new webpack.optimize.ModuleConcatenationPlugin();` to my webpack build and was very surprised that it had no effect. [It took me a while](new webpack.optimize.ModuleConcatenationPlugin()
https://github.com/aurelia/webpack-plugin/issues/153#issuecomment-420376814) to realize that it is auto-enabled by default in production.

I’ve added a corresponding note to prevent others from making the same mistake.

### CODE REVIEW

File: src/content/plugins/module-concatenation-plugin.md [hunk 1]
```diff
@@ -10,12 +10,13 @@ related:
 
 In the past, one of webpack’s trade-offs when bundling was that each module in your bundle would be wrapped in individual function closures. These wrapper functions made it slower for your JavaScript to execute in the browser. In comparison, tools like Closure Compiler and RollupJS ‘hoist’ or concatenate the scope of all your modules into one closure and allow for your code to have a faster execution time in the browser.
 
-This plugin will enable the same concatenation behavior in webpack.
+This plugin will enable the same concatenation behavior in webpack. It is [enabled by default in production](/configuration/optimization/#optimization-concatenatemodules) but you can also manually add it:
```
Comment (montogeek) [inline src/content/plugins/module-concatenation-plugin.md]: Uhmm, it is better to avoid plugin duplication.

Comment (rluba) [reply src/content/plugins/module-concatenation-plugin.md]: One more reason to document the default :-)
What kind of wording would you prefer?
"It is enabled by default in production but you can add it manually if you need it in other circumstances"?

Comment (montogeek) [reply src/content/plugins/module-concatenation-plugin.md]: "...in other mode".

File: src/content/plugins/module-concatenation-plugin.md [hunk 2]
```diff
@@ -10,12 +10,13 @@ related:
 
 In the past, one of webpack’s trade-offs when bundling was that each module in your bundle would be wrapped in individual function closures. These wrapper functions made it slower for your JavaScript to execute in the browser. In comparison, tools like Closure Compiler and RollupJS ‘hoist’ or concatenate the scope of all your modules into one closure and allow for your code to have a faster execution time in the browser.
 
-This plugin will enable the same concatenation behavior in webpack.
+This plugin will enable the same concatenation behavior in webpack. It is [enabled by default in production](/configuration/optimization/#optimization-concatenatemodules) but you can also add it manually if you need it in other modes:
```
Comment (EugeneHlushko) [inline src/content/plugins/module-concatenation-plugin.md]: The thing is, linked page already says that this optimization is enabled by default in `production` mode. 
```
Depends on `optimization.providedExports` and `optimization.usedExports`. By default `optimization.concatenateModules` is enabled in production mode and disabled elsewise.
```

Also if you go to mode page: https://webpack.js.org/concepts/mode/#usage is shows you which plugins are enabled by default in production mode.

If you still feel like adding it here, please use the same phrasing as we have:

```
By default this plugin is already enabled in production [`mode`](/concepts/mode/#mode-production) and disabled elsewise.
```

**Comment by montogeek** [comment]
@rluba Friendly ping

File: src/content/plugins/module-concatenation-plugin.md (hunk 2 above)
Comment (rluba) [reply src/content/plugins/module-concatenation-plugin.md]: Done

**Comment by rluba** [comment]
@montogeek Thanks for the ping. I’ve updated the wording.

**Comment by montogeek** [comment]
@EugeneHlushko Please take another look

**Comment by EugeneHlushko** [comment]
Thanks!

**Pull request merged**